ingestion/src/metadata/sampler/sqlalchemy/sampler.py

```diff
@@ -230,7 +229,7 @@ def _rdn_sample_from_user_query(self) -> Query:
 
     def _partitioned_table(self) -> Query:
         """Return the Query object for partitioned tables"""
-        return aliased(self.raw_dataset, self.get_partitioned_query().subquery())
+        return self.get_partitioned_query()
```

Comment:
Why is this method required? I suppose we can just use `self.get_partitioned_query()`